src/main.rs

```diff
@@ -51,6 +53,8 @@ fn execute_instruction(instruction: &str, arguments: &str, cpu: &mut Cpu, ram: &
             println!("The a register is: {a}");
             set_zero_flag(cpu.a_register, cpu);
             set_negative_flag(cpu.a_register, cpu);
+            clear_carry_flag(cpu);
```

Comment:
Let's leave this here for testing purposes. But we need to remove once we add the operation.